image/manifest_test.go

```diff
@@ -122,3 +103,45 @@ func TestUnpackLayer(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+func createTarFile(name string, list []tarContent) (descriptor, error) {
```

Comment:
And we have a public `Descriptor` type in `github.com/opencontainers/image-spec/specs-go`.  Can we return that instead of the private, local `descriptor` (one step closer to being able to drop the local `descriptor` ;)?